src/Tribe/Bar.php

```diff
@@ -37,17 +39,16 @@ public function __construct() {
 	 *
 	 */
 	public function should_show() {
-
 		if ( ! $wp_query = tribe_get_global_query_object() ) {
 			return;
 		}
 
-		$show_bar_filter = in_array(
-			get_post_type(), array(
-				Tribe__Events__Main::VENUE_POST_TYPE,
-				Tribe__Events__Main::ORGANIZER_POST_TYPE,
-			)
-		) ? false : true;
+		$not_in_post_types = array(
```

Comment:
I think this variable should be called `$allowed_types` as the `not` is applied when using the `!` operator during the comparison on `in_array`